packages/exo/README.md

```diff
@@ -10,13 +10,19 @@ When an exo is defined with an InterfaceGuard, the exo is augmented by default w
 
 ```js
 // `GET_INTERFACE_GUARD` holds the name of the meta-method
-import { GET_INTERFACE_GUARD } from `@endo/exo`;
+import { GET_INTERFACE_GUARD } from '@endo/exo';
+import { getCopyMapKeys } from '@endo/patterns';
 
 ...
    const interfaceGuard = await E(exo)[GET_INTERFACE_GUARD]();
    // `methodNames` omits names of automatically added meta-methods like
    // the value of `GET_INTERFACE_GUARD`.
    // Others may also be omitted if `interfaceGuard.partial`
```

Comment:
Attn @michaelfig 

My mistake! In anything that saw the light of day, it was never called `interfaceGuard.partial`. It became `interfaceGuard.sloppy`, but I forgot to rename this occurrence. 

But @michaelfig is in process with a PR that will make changes in this area, so leave that to him. For your PR, no change suggested.